quodlibet/util/collection.py

```diff
@@ -47,19 +48,73 @@
 ELPOEP = list(reversed(PEOPLE))
 PEOPLE_SCORE = [100 ** i for i in range(len(PEOPLE))]
 
+SMOOTHING_FACTOR_DEFAULT = 10
 
-def avg(nums):
+
+def unweighted_average(nums):
     """Returns the average (arithmetic mean) of a list of numbers"""
     return float(sum(nums)) / len(nums)
 
 
-def bayesian_average(nums, c=None, m=None):
+def unweighted_bayesian_average(nums, c=None, m=None):
     """Returns the Bayesian average of an iterable of numbers,
     with parameters defaulting to config specific to ~#rating."""
     m = m or config.RATINGS.default
     c = c or config.getfloat("settings", "bayesian_rating_factor", 0.0)
-    ret = float(m * c + sum(nums)) / (c + len(nums))
-    return ret
+    return float(m * c + sum(nums)) / (c + len(nums))
+
+
+def hyperbolic_smoother(x, n=SMOOTHING_FACTOR_DEFAULT):
```

Comment:
I'd say (in light of above around two places for default), just make `n` mandatory, then we don't need the default here too

Reply:
Makes sense!